x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_creation/components/create_step/progress_stats.tsx

```diff
@@ -44,6 +53,12 @@ export const ProgressStats: FC<{ jobId: DataFrameAnalyticsId }> = ({ jobId }) =>
 
         if (jobStats !== undefined) {
           const progressStats = getDataFrameAnalyticsProgressPhase(jobStats);
+
+          if (jobStats.state === FAILED) {
+            clearInterval(interval);
+            setFailedJobMessage(jobStats.failure_reason || `Analytics job ${jobId} has failed.`);
```

Comment:
Does this text need internationalizing?

Reply:
Good catch - updated here 1c4a92582305559d75dc20961987e00af4ff7579